torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -122,7 +141,7 @@ def _lower_to_native_backend(model: QuantizedGraphModule) -> QuantizedGraphModul
     to the native backend in PyTorch (fbgemm/qnnpack), both backends shares the same
     operator signature so they can be lowered with the same function
     """
-    for ref_class in LOWER_MODULE_MAP.keys():
+    for ref_class in list(LOWER_MODULE_MAP.keys()) + list(LOWER_FUSED_MODULE_MAP):
```

Comment:
also, I'm wondering if we can just move this for loop to `_lower_weighted_ref_module` itself?